### PR TITLE
Add Moesif pump for logging API analytics

### PR DESCRIPTION
**Pull request by dgilling**
Add pumps/moesif.go, update README config example
@lonelycode Please review.

### CODE REVIEW

File: pumps/moesif.go [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+
+	return nil
+}
+
+func (p *MoesifPump) WriteData(data []interface{}) error {
+	log.WithFields(logrus.Fields{
+		"prefix": moesifPrefix,
+	}).Info("Writing ", len(data), " records")
+
+	if len(data) > 0 {
```
Comment (mvdan) [inline pumps/moesif.go]: return early and unindent the rest

File: pumps/moesif.go [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+		return nil, fmt.Errorf("Error while splitting raw data")
+	}
+
+	headers := decodeHeaders(&headersBody[0])
+
+	var body interface{}
+	if len(headersBody) == 2 {
+		body = base64.StdEncoding.EncodeToString([]byte(headersBody[1]))
+	} else {
+		body = nil
```
Comment (mvdan) [inline pumps/moesif.go]: dead assignment

File: pumps/moesif.go [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+}
+
+func decodeRawData(raw string) (*RawDecoded, error) {
+	headersBody := strings.SplitN(raw, "\r\n\r\n", 2)
+
+	if len(headersBody) == 0 {
+		return nil, fmt.Errorf("Error while splitting raw data")
+	}
+
+	headers := decodeHeaders(&headersBody[0])
```
Comment (mvdan) [inline pumps/moesif.go]: why are you using a pointer to a string?

**Comment by buger** [comment]
Can you also please add small description about Moesif to Readme and link to the service as well. Thank you for contribution!

**Comment by dgilling** [comment]
Thanks for the review. Changes pushed.

**Pull request merged**